app/assets/javascripts/leaflet.map.js

```diff
@@ -55,6 +55,13 @@ L.OSM.Map = L.Map.extend({
 
     this.dataLayer = new L.OSM.DataLayer(null);
     this.dataLayer.options.code = 'D';
+    
+    this.gpsLayer = new L.OSM.GPS({
+      attribution: copyright,
+      code: "G",
+      keyid: "gps",
```

Comment:
just one question: is the attribution key needed here? If I add it (as it currently is) I see the copyright twice one I check that box